### DISCUSSION TITLE
Repost: Pivot from Grid to Grafana

### DISCUSSION THREAD

**Discussion by darth-drew** [Security-Onion-Solutions/securityonion#3711] [Ideas] [answered]
Bringing this up again to see if others are interested.

For our sysadmins spinning up new nodes, it would be cool for them to be able to click on a new line in the sensors interface (similar to the chevron in the alerts interface) and jump to the corresponding grafana dashboard, in order to quickly check on health and status.

Thanks for the idea. We will take this into consideration.

_Originally posted by @dougburks in https://github.com/Security-Onion-Solutions/securityonion/discussions/1966#discussioncomment-132602_

**Comment by dougburks** [comment, marked as answer]
We shared more information about Security Onion 2.4 at our conference last week and mentioned this specific feature:
https://blog.securityonion.net/2022/10/sneak-peek-at-security-onion-24.html

**Reply by darth-drew** [reply]
I saw! Looking forward to it! Great work!
